docker/puppetserver-standalone/docker-entrypoint.sh

```diff
@@ -1,51 +1,9 @@
 #!/bin/bash
 
-chown -R puppet:puppet /etc/puppetlabs/puppet/
-chown -R puppet:puppet /opt/puppetlabs/server/data/puppetserver/
-
-# During build, pristine config files get copied to this directory. If
-# they are not in the current container, use these templates as the
-# default
-TEMPLATES=/var/tmp/puppet
-
-cd /etc/puppetlabs/puppet
-for f in auth.conf hiera.yaml puppet.conf puppetdb.conf
-do
-    if ! test -f $f ; then
-        cp -p $TEMPLATES/$f .
-    fi
+for f in /docker-entrypoint.d/*.sh; do
+    echo "Running $f"
+    chmod +x "$f"
+    "$f"
```

Comment:
Should we have a `set -e` at the top of the script to fail fast? We probably want any erroring script to kill the whole entrypoint and never allow healthy, right?

Reply:
Good idea ! I updated the PR accordingly